### DISCUSSION TITLE
Where to find and how to see the downloaded pictures

### DISCUSSION THREAD

**Discussion by Harmonicgears** [TumblThreeApp/TumblThree#505] [Q&A (Question & Answer)]
Hello.

I am sadly very much a noob when it comes to use free software. 

While the Application said, it downloaded all the files i cant seem to find the downloaded images.

Can you please help me out where i need to look for them?

And how do i make sure that they are actually downloaded ?

**Comment by thomas694** [comment]
Hello,
you can find the downloaded files in the blog's download folder. One way to open it is to right-click on the blog and choose `Show files`.

**Reply by Harmonicgears** [reply]
Hello

I tried to do the basic trouble shooting but despite the Prorgamm telling me, it downloaded everything successfully, i still couldnt find the files in the set download folder.
After testing TumblrThree on a friend PC, where it worked right away with a set download folder, I tried it again with my PC

What ended up working was following

The download folder was set to a directory on the **public profile**/account of my PC. After the download finished, all the files and pictures where there as expected.


Well, I hope that helps you all somewhat
